app/models/state_registrants/wa_registrant.rb

```diff
@@ -286,13 +287,9 @@ def to_wa_data
       "driverLicense"	=>	self.driver_license.to_s.gsub(/\s-/, ''),
   
       
-      "issue_date" =>self.issue_date.iso8601, #CTW check foramt
-      "SSN4"	=>	self.ssn4, #CTW Check
-      
-      #"DLE"	=>	self.unknown,  Ask Alex
-      #"oldDLN"	=>	self.unknown,  Ask Alex
-
-  
+      "issueDate" =>self.issue_date.strftime("%m/%d/%Y"), #.iso8601, #CTW check foramt
+      "SSN4"	=>	self.ssn4.blank? ? nil:self.ssn4, #CTW Check
```

Comment:
I assume this works as is, but generally you see spaces around the : so
`self.ssn4.blank? ? nil : self.ssn`